game_eggs/FoundryVTT/egg-FoundryVTT.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-11-13T13:06:21-08:00",
+    "name": "Foundry VTT",
+    "author": "pterodactyl@demi.cloud",
+    "description": "Foundry VTT is a standalone application built for experiencing multiplayer tabletop RPGs using a feature-rich and modern self-hosted application where your players connect directly through the browser.",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:nodejs_17"
+    ],
+    "file_denylist": [],
+    "startup": "node .\/app\/resources\/app\/main.js --dataPath=\"\/home\/container\/data\"",
+    "config": {
+        "files": "{\r\n    \"data\/Config\/options.json\": {\r\n        \"parser\": \"json\",\r\n        \"find\": {\r\n            \"port\": \"{{server.build.default.port}}\",\r\n            \"language\": \"{{server.build.env.FOUNDRY_LANGUAGE}}\",\r\n            \"updateChannel\": \"{{server.build.env.UPDATE_CHANNEL}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"Server started and listening on port\"\r\n}",
+        "logs": "{}",
+        "stop": "^c"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# FoundryVTT install script\r\n#\r\n# Server Files: \/mnt\/server\r\ndeclare -r DIR_ROOT=\"\/mnt\/server\"\r\ndeclare -r DIR_APP=\"${DIR_ROOT}\/app\"\r\ndeclare -r DIR_DATA=\"${DIR_ROOT}\/data\"\r\ndeclare -r ZIP_FILE_NAME=\"foundryvtt.zip\"\r\n\r\nmain() {\r\n  apt update\r\n  apt install -y unzip\r\n\r\n  printf \"\\nBuilding directory structure...\\n\"\r\n  mkdir -p \"${DIR_ROOT}\/data\/Config\"\r\n  mkdir -p \"${DIR_ROOT}\/app\"\r\n  # shellcheck disable=SC2164\r\n  cd \"${DIR_APP}\"\r\n  printf \"\\nDownloading FoundryVTT files...\\n\"\r\n  wget \"${TIMED_URL}\" -O \"${ZIP_FILE_NAME}\"\r\n  printf \"\\nunzipping FoundryVTT files...\\n\"\r\n  unzip \"${ZIP_FILE_NAME}\" -d \"${DIR_APP}\"\r\n  rm \"${ZIP_FILE_NAME}\"\r\n\r\n  printf \"\\nGenerating default configuration...\\n\"\r\n  cat <<EOF >\"${DIR_DATA}\/Config\/options.json\"\r\n{\r\n  \"port\": 30000,\r\n  \"upnp\": false,\r\n  \"fullscreen\": false,\r\n  \"hostname\": null,\r\n  \"localHostname\": null,\r\n  \"routePrefix\": null,\r\n  \"sslCert\": null,\r\n  \"sslKey\": null,\r\n  \"awsConfig\": null,\r\n  \"dataPath\": \"\/home\/container\/data\",\r\n  \"passwordSalt\": null,\r\n  \"proxySSL\": false,\r\n  \"proxyPort\": null,\r\n  \"minifyStaticFiles\": true,\r\n  \"updateChannel\": \"release\",\r\n  \"language\": \"en.core\",\r\n  \"upnpLeaseDuration\": null,\r\n  \"world\": null\r\n} \r\nEOF\r\n  printf \"Installation Done.\\n\"\r\n}\r\nmain \"@\"",
```

Comment:
If you made this on windows it may have added the \r and that may cause issues.

Reply:
huh, I thought that I set all my editors to use linux newline endings. I'll double-check this when I have a chance. thanks!